java/ql/integration-tests/all-platforms/kotlin/logs/test.py

```diff
@@ -1,4 +1,39 @@
-from create_database_utils import *
+import json
+from math import log
+import pathlib
+import re
 
-run_codeql_database_create(['kotlinc test.kt'], test_db="kt-db", db=None, lang="java")
-run_codeql_database_create(['"%s" ./index_logs.py' % sys.executable], lang="java")
+
+def test(codeql, java_full, cwd: pathlib.Path, expected_files):
+    codeql.database.create(command=["kotlinc test.kt"])
+    expected_files.add("logs.actual")
+
+    with open("logs.actual", "w") as f_out:
+        log_dir = cwd / "test-db" / "log"
+        file_index = 0
+        for log_file in log_dir.glob("kotlin-extractor*.log"):
+            file_index += 1
+            f_out.write(f"Test script\nLog file\n{file_index}\n")
```

Comment:
as far as I can understand, the `Test script` and `Log file` strings were there just to accomodate the QL query table format, so I'd just go with
```suggestion
            f_out.write(f"Log file {file_index}\n")
```